slapr/slack.py

```diff
@@ -3,6 +3,8 @@
 
 import slack
 
+URL_PATTERN = r"<(?P<url>.*)>"
```

Comment:
Naming nit (to be consistent with the `pr_url_pattern` parameter):

```suggestion
DEFAULT_PR_URL_PATTERN = r"<(?P<url>.*)>"
```